api/tests/unit/features/versioning/test_unit_versioning_views.py

```diff
@@ -1664,6 +1665,20 @@ def test_list_versions_returns_all_versions_for_enterprise_plan(
     mocker.patch("organisations.models.is_saas", return_value=is_saas)
     mocker.patch("organisations.models.is_enterprise", return_value=not is_saas)
 
+    if is_saas is False:
+        licence_content = {
+            "organisation_name": "Test Organisation",
+            "plan_id": "Enterprise",
+            "num_seats": 20,
+            "num_projects": 3,
+            "num_api_calls": 3_000_000,
+        }
+
+        OrganisationLicence.objects.create(
+            organisation=environment_v2_versioning.project.organisation,
+            content=json.dumps(licence_content),
+        )
```

Comment:
This makes me think that we should split this case out from this parametrised test.

Reply:
Sounds good.